thoth/common/openshift.py

```diff
@@ -1683,6 +1687,22 @@ def schedule_adviser(
             },
         )
 
+    def _assign_workflow_parameters(self) -> Dict[str, Any]:
+        """Check and assign workflow parameters for different services."""
+        self.ceph_bucket_prefix = os.environ["THOTH_CEPH_BUCKET_PREFIX"]
+        self.ceph_bucket_name = os.environ["THOTH_CEPH_BUCKET"]
+        self.ceph_host = urlparse(os.environ["THOTH_S3_ENDPOINT_URL"]).netloc
+        self.deployment_name = os.environ["THOTH_DEPLOYMENT_NAME"]
```

Comment:
OpenShift class does not have these properties, assign them directly to the dict retuned:

```
        return {
            "ceph_bucket_prefix": os.environ["THOTH_CEPH_BUCKET_PREFIX"],
     ...
```